### DISCUSSION TITLE
i cant proceed to Step to in the education/Primer-copilot

### DISCUSSION THREAD

**Discussion by arturoCrisanto** [community/community#123747] [GitHub Education] [answered]
### Hi everyone,

i cant proceed to the next step of this tutorial in github education, its about the primer-copilot 

i already waited 60 seconds and refreshed it but nothing happened, i hope some explained it to me

<img width="832" alt="Capture" src="https://github.com/community/community/assets/103894948/cf4cfb7c-4242-40b5-924d-994e3cd7c1b9">

**Comment by gitstua** [comment]
👋🏽  Hi @arturoCrisanto Thanks for being part of our GitHub Community.

One of the great things about GitHub is you can find others who are working on similar things. Did you create your `devcontainer.json` file similar to this one from another one of our community? [github.com/cyz/skills-copilot-codespaces-vscode/blob/main/.devcontainer/devcontainer.json](https://github.com/cyz/skills-copilot-codespaces-vscode/blob/main/.devcontainer/devcontainer.json)

There could be a syntax error maybe in your version of this file?

**Reply by arturoCrisanto** [reply]
thank you ill check on this

**Comment by gitstua** [comment, marked as answer]
I just worked through this and here is a video of how it worked for me... 
![copilot-skills](https://github.com/community/community/assets/25424433/9ac617ca-20dc-4dd9-a2d4-7c12f8a6fd47)

**Reply by Stormlocc** [reply]
The error is in the 1st step, don't load the other files, only be /.devcontrainer/devcontainer.json
